package.json

```diff
@@ -7,6 +7,8 @@
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
     "test:e2e": "vue-cli-service test:e2e",
+    "test:cypress": "cypress run",
```

Comment:
what does this task do? It doesn't run standalone

Reply:
It could but easier to have everything starting directly from yarn

Reply:
ok, I think I see what you mean now. Maybe put a comment on this.